cli/internal/cmd/root.go

```diff
@@ -25,10 +25,21 @@ var (
 
 func recovery() {
 	if r := recover(); r != nil {
-		fmt.Println(emoji.Sprintf("\n:skull:Oops! Unexpected error occurred. Please contact tech support.\n"))
+		// Always log to file
 		log.Error("panic detected", "error", r)
 		log.Error(string(debug.Stack()))
```

Comment:
This panic log is not properly formatted which means it won't be formatted properly on the UI to the customers as well. Should we fix the formatting on the logged stack trace ?

Reply:
sure, what kind of formatting were you thinking of?****